src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,51 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+    private static final int DATE_INDEX = 0;
+    private static final int NAME_INDEX = 1;
+    private static final int WORK_HOUR_INDEX = 2;
+    private static final int INCOME_PER_HOUR_INDEX = 3;
+    private static final String DIVIDER = " ";
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        int sumSalary;
+        int hour;
+        int salary;
```

Comment:
There is no point to declare bunch of variables way before they are used.
most of them may live only inside some loop or if block
```suggestion
```